### DISCUSSION TITLE
Does Windows in S mode have any benefits?

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#419] [Q&A] [answered]
https://support.microsoft.com/en-gb/windows/windows-10-and-windows-11-in-s-mode-faq-851057d6-1ee9-b9e5-c30b-93baebeebc85

**Comment by HotCakeX** [comment, marked as answer]
Sure, it reduces the attack surface by a lot. You can enable S mode using App Control policies on any system. According to the link you shared, S mode only allows Microsoft store apps to be used, this restriction can be configured via [AppControl Manager](https://github.com/HotCakeX/Harden-Windows-Security/wiki/AppControl-Manager). Let me know if you need help with that.
